docs/guide/content/installation.md

```diff
@@ -1,64 +1,95 @@
 
-## System Requirements
+!!! note "Development Notes"
 
-OnDemand Loop targets the same software versions used by Open OnDemand itself.
-Use the versions below (or the versions bundled with your OOD release):
+    This guide covers production installation only. If you're setting up OnDemand Loop for development purposes or as an Open OnDemand development application, please refer to the [Development Guide](development_guide/index.md) instead.
+
+
+### System Requirements
+
+The software versions you need depend on which version of Open OnDemand you're running.
+OnDemand Loop is designed and tested to work with the Ruby and Node.js versions bundled with your Open OnDemand installation.  
+Here's what you need based on your OOD version:
+
+**For Open OnDemand 3.x:**
+
+- Ruby 3.1
+- Node.js 18
+
+**For Open OnDemand 4.x:**
+
+- Ruby 3.3  
+- Node.js 20
+
+#### Required Software
 
-- **Ruby** 3.1
-- **Node.js** 18
 - **Open OnDemand** 3.1 or newer
+- **Ruby** 3.1 or 3.3 (matching your OOD version)
+- **Bundler** (usually comes pre-installed with Ruby)
+- **Node.js** 18 or 20 (matching your OOD version)
 
-## Building the Application
+#### Finding the Right Versions
 
-The repository includes a Makefile with helper targets. The preferred way to
-compile Loop is to install all Ruby gems and Node packages into local folders,
-avoiding conflicts with system-wide installs. This is exactly how the
-`scripts/loop_build.sh` script operates and how the application has been tested.
+If you're unsure which versions to use, check the bundled software that came with your Open OnDemand installation.
+OnDemand Loop includes a version mapping matrix used for the local environment [ood_versions.mk](https://github.com/IQSS/ondemand-loop/tools/make/ood_versions.mk)
+that shows exactly which Ruby and Node.js versions correspond to each OOD release.
+
+### Building the Application
+
+<a id="ondemand-loop-repo-info"></a>
+!!! info ":fontawesome-brands-github: OnDemand Loop Repo"
+
+     - [https://github.com/IQSS/ondemand-loop](https://github.com/IQSS/ondemand-loop)
+     - [https://github.com/IQSS/ondemand-loop/releases](https://github.com/IQSS/ondemand-loop/releases)
+
+The repository includes a Makefile with helper targets and utility scripts to facilitate the building process.
+The preferred way to build OnDemand Loop is to install all Ruby gems and Node packages into local folders, avoiding conflicts with system-wide installs. 
+This is exactly how the [scripts/loop_build.sh](https://github.com/IQSS/ondemand-loop/scripts/loop_build.sh) script operates and how the application has been tested.
 See that script for the exact commands executed during the build.
-To run the build inside the Docker based builder image, execute:
+
+To run the build inside the default Docker builder image targeting an OnDemand v3.1.7 installation, execute:
 
```

Comment:
I miss another `make relesase_build` example after this explaining how to specify an OnDemand version, for example to install in version 4.x